### ISSUE TITLE
documentation improvement request

### ISSUE THREAD

**Issue by duaneellissd** [gnu-mcu-eclipse/gnu-mcu-eclipse.github.io-source#10]
This link describes the location where "packs" are stored, it's not obvious how to find this dialog in Eclipse...

Can you describe where/how/Navigation path to find this window in eclipse:

https://gnu-mcu-eclipse.github.io/plugins/packs-manager/#configuration

**Comment by ilg-ul** [comment]
right, thank you, there were some changes in this respect, and the documentation needs some updates.
